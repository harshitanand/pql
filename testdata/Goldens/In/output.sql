SELECT * FROM "StormEvents" WHERE "State" IN ('GEORGIA', 'MISSISSIPPI');
