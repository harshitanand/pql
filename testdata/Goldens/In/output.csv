11503,GEORGIA,Thunderstorm Wind,2000
13913,MISSISSIPPI,Thunderstorm Wind,20000
